.travis.yml

```diff
@@ -20,14 +20,16 @@ env:
       - OS=fedora DIST=26
       - OS=fedora DIST=27
 
-install:
-  - sudo apt-get install coreutils
-
 script:
+  # lulpeg tags have the following format: major.minor
+  # They are not annotated and so can not be used by packpack
+  # automaticaly. Also packpack excpects major.minor.patch tag format.
+  # That is why we have to pass VERSION and RELEASE.
   - git describe --tags
-  - VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | sed 's#-#\.#g' | cut -c 2-`
   - git clone https://github.com/packpack/packpack.git packpack;
-  - VERSION=$VERSION packpack/packpack;
+  - VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | cut -c 2- | cut -d '-' -f-1`
```

Comment:
`-f1` would be more intuitive then `-f-1`.

Reply:
`sed 's/\(.*\)-.*/\1/' | cut -c 2-` can be written as `sed 's/.\(.*\)-.*/\1/'`

And more. You can use just sed. Like so:

```
git describe --tags | sed 's/^v\([0-9.]\+\)-.*$/\1/'  # ver
git describe --tags | sed 's/^.*-\([0-9]\+\)-.*$/\1/'  # rel
```

Reply:
Remove RELEASE at all as it is set by packpack to 1 by default and we are ok with it. Simplified sed expression, removed `cut` usage.